docs/en/reference/HTTP_API/SQL.md

```diff
@@ -0,0 +1,189 @@
+---
+displayed_sidebar: "English"
+---
+
+# HTTP SQL API
+
+## Description
+
+StarRocks v3.2.0 introduces the HTTP SQL API for users to perform various types of queries using HTTP. Currently, this API supports SELECT, SHOW, EXPLAIN, and KILL statements.
+
+Syntax using the curl command:
+
+```shell
+curl -X POST 'http://<fe_ip>:<fe_http_port>/api/v1/catalogs/<catalog_name>/databases/<database_name>/sql' \
+   -u '<username>:<password>'  -d '{"query": "<sql_query>;", "sessionVariables":{"var_name":<var_value>}}' \
+   --header "Content-Type: application/json"
+```
+
+## Request message
+
+### Request line
+
+```sql
+POST 'http://<fe_ip>:<fe_http_port>/api/v1/catalogs/<catalog_name>/databases/<database_name>/sql'
+```
+
+| Field                    | Description                                                  |
+| ------------------------ | :----------------------------------------------------------- |
+|  fe_ip                   | FE node IP address.                                                  |
+|  fe_http_port            | FE HTTP port.                                           |
+|  catalog_name            | The catalog name. Currently, this API supports querying only internal tables, which means `<catalog_name>` can only be set to `default_catalog`. |
+|  database_name           | The database name. If no database name is specified and a table is used in the SQL query, you must prefix the table name with its database name, for example, `database_name.table_name`. |
```

Comment:
If ~~no database name is specified and~~ a table is used in the SQL query, you must prefix the table name with its database name, for example, `database_name.table_name`.